themes/frontierline/style.css

```diff
@@ -18,729 +18,3023 @@ Domain Path: /languages/
 *
 * Don't edit style.css directly!
 **********/
-@font-face { font-family: 'Zilla Slab'; font-weight: normal; font-style: normal; src: url("fonts/ZillaSlab-Regular.woff2") format("woff2"), url("fonts/ZillaSlab-Regular.woff") format("woff"); }
```

Comment:
The diff is huge. Try `sass --style compact --sourcemap=auto ./sass/style.scss style.css` in the theme root. @craigcook can you add the correct sass command in the readme, please? I am not sure if this one is the best to use either.

Reply:
Correct, it's compact style.

http://sass-lang.com/documentation/file.SASS_REFERENCE.html#output_style